app/lib/konnectors.js

```diff
@@ -6,7 +6,7 @@ const STATE_READY = 'ready'
 export function addAccount (cozy, konnector, account) {
   if (!konnector.accounts) konnector.accounts = []
   konnector.accounts.push(account)
-  return Promise.resolve(konnector)
+  return Promise.resolve(account)
```

Comment:
This will break the modal when adding a new account, this new account won't be added to the list since the konnector is not returned anymore. See https://github.com/cozy/cozy-my-accounts/blob/master/app/containers/ConnectorManagement.jsx#L153